packages/bson-transpilers/test/index.test.js

```diff
@@ -0,0 +1,34 @@
+const assert = require('assert');
+const transpilers = require('../index');
+
+const SAMPLE = transpilers.javascript.java;
+
+const VALID_OPTIONS = {
+  uri: 'mongodb://localhost',
+  database: 'test',
+  collection: 'webscale'
+};
+
+const INVALID_JS = '{ ... }';
+const VALID_JS = '({ a : 1 })';
+
+describe('bson transpiler', function() {
```

Comment:
bson-transpiler tests are usually added as part of existing yaml templates, they also already set up to run tests against _all_ languages with clear expectation template, why are we breaking the pattern here?

Reply:
Because this test is not testing the output template, it's testing a validation that happens earlier, and can not be tested using the yaml mechanism.

Reply:
We are changing comilation output based on the input parameter, why are we not testing it and can't add those tests to the existing testing setup?

Reply:
There are tests for that, but I'm also testing the validation, as the bson-transpilers will try to compile each of the parameters passed in the input object but those that are in the test. If they are not valid javascript, it will throw an exception (which happens) so they should be tested separately.